test/unit/math/opencl/prim/add_test.cpp

```diff
@@ -1,225 +1,71 @@
 #ifdef STAN_OPENCL
 #include <stan/math/prim.hpp>
-#include <stan/math/opencl/prim.hpp>
+#include <test/unit/math/opencl/util.hpp>
 #include <test/unit/util.hpp>
 #include <gtest/gtest.h>
 #include <algorithm>
 
-TEST(MathMatrixCL, add_v_exception_pass) {
-  stan::math::vector_d d1, d2;
+auto add_functor
+    = [](const auto& a, const auto& b) { return stan::math::add(a, b); };
+auto plusop_functor = [](const auto& a, const auto& b) { return a + b; };
```

Comment:
operator+ is already tested in kernel_generator tests. Do these tests add anything kg tests are missing?

Reply:
No, this doesn't add anything. The operator+ tests were here previously so I just moved them to use the util function. Will remove.